db/data/board_games.js

```diff
@@ -0,0 +1,18 @@
+const apiCalls = require("../../apiCalls.js")
+
+const board_gameData = async () => {
```

Comment:
I have been using camelCase to name my function, should we write all our `server-side` code in sanake_case and `front-end` camelCase?

Reply:
I am not sure honestly, this looks a bit contrived to me, and seems like camelCase would be just fine.

Reply:
I have been using camelCase for everything except the names of tables and columns for the database, just FYI!